src/main/java/session_17_equals_hashcode_generics/challenge/challenge_7/Entity.java

```diff
@@ -0,0 +1,40 @@
+package session_17_equals_hashcode_generics.challenge.challenge_7;
+
+import java.util.Objects;
+
+public class Entity {
+    private int id;
```

Comment:
Please don't forget to add an empty row below the class declaration, and id can be a long instead of int.